src/scss/design-system-interface.scss

```diff
@@ -0,0 +1,42 @@
+.drizzle-variant-set {
```

Comment:
I don't love these names, because the words "set" and "group" are so close.

A "set" contains _all_ a pattern's variants. Basically the root to contain all variant stuff.

A "group" is a collection of mutually exclusive variants. You can only pick one variant in a group at a time.

Better name ideas are very welcome. :)

Reply:
They work for me 🤷🏻‍♀️